src/ttar/ttar

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
```

Comment:
do we want this file in the vault-boshrelease codebase, or can we rely on the version we pull down from github in the rake task?

Reply:
I made the rake task just to help maintain the repo, we need it somewhere, either pulling from the same store as the vault binary or stored in the codebase.

Reply:
The Rakefile is probably overkill.  ttar is feature-complete and done.  All bugs found now are considered to be features.

Reply:
Yea I kinda figured that. Since it won't really be changing it would be nice if we could get it into the community S3 bucket then we can just load it here with everything else: https://github.com/cloudfoundry-community/vault-boshrelease/blob/master/config/blobs.yml

We only need it at runtime and could reference it from `/var/vcap/packages/...`

Reply:
Or is there some way to use that configuration to pull from GitHub?

Reply:
Is there a reason it can't just go in `src/ttar/ttar`?

Reply:
/shrug not really just being all particular. It's the only thing in there and really has nothing directly to do with Vault but I don't think I'll loose too much sleep over it. Are we good to merge given the current state?